client/blocks/category-pill-navigation/README.md

```diff
@@ -0,0 +1,68 @@
+# Category Pill Navigation
+
+This navigation component, designed for consistent design integration, is primarily used on the `/patterns/:category`, `/themes`, and `/blog` pages. It offers a uniform design aesthetic across various sections of the website.
+
+## How to use
+
+```js
+import { CategoryPillNavigation } from 'calypso/components/category-pill-navigation';
+import ImgStar from 'calypso/my-sites/patterns/pages/category/images/star.svg';
+
+function render() {
+	return (
+		<CategoryPillNavigation
+			list={ Array.from( { length: 15 }, ( _, i ) => ( {
+				name: `category-${ i }`,
+				label: `Category ${ i }`,
+				link: '#',
+			} ) ) }
+			selectedCategory="category-2"
+			buttons={ [
+				{
+					icon: ImgStar,
+					label: 'Discover',
+					link: '/',
+				},
+				{
+					icon: ImgStar,
+					label: 'All categories',
+					link: '/',
+				},
+			] }
+		/>
+	);
+}
+```
+
+## Props
+
+Below is a list of supported props.
+
+### `list`
+An array of objects, each representing a navigation link:
+
+```
+{
+	name: string; // Acts as an `id` and is primarily used for highlighting the active item.
```

Comment:
If it's supposed to be an identifier, then maybe we should rename it to `id`, `slug` or `key`? It's always ambiguous to have two fields named `name` and `label` side-by-side.

Reply:
We receive it from backend, so I would keep it as is, instead of renaming the key.